DeAceroDLabs/marketplace-frontend
# Feature/dmp 432

## What is done

[[DMP-432] Navigation arrows on categories](https://deacero.atlassian.net/browse/DMP-432)

## How is done

1. Aligned footer and header.
2. Added navigation arrows in home.
3. Added active prop to button.
4. Added loadingCategories state to show arrows or not.

## How to test

1. Run the project via `npm start`.
2. login
3. There you will see the navigations arrows.
